packages/passport-auth/lib/Passport.js

```diff
@@ -44,7 +49,7 @@ class PassportAuthStrategy {
       this.keystone.createList(this.config.sessionListKey, {
         fields: {
           [FIELD_SERVICE_NAME]: { type: Text },
-          [FIELD_USER_ID]: { type: Text },
+          [FIELD_USER_ID]: { type: Text, mongooseOptions: { index: true } },
```

Comment:
This will only add an index for Mongoose, but not for any other database adapter (eg; Postgres).

Reply:
we should have universal index option and have documentation if the adapter supports it or not

essentially `index: true` on field rather than mongooseOptions or knexOptions

Reply:
Brilliant idea, I've captured that in this issue: https://github.com/keystonejs/keystone-5/issues/1297